django/test/testcases.py

```diff
@@ -524,17 +524,23 @@ def assertContains(
             response, text, status_code, msg_prefix, html
         )
 
+        if response.streaming:
+            content = b"".join(response.streaming_content)
+        else:
+            content = response.content
+
         if count is not None:
             self.assertEqual(
                 real_count,
                 count,
                 msg_prefix
-                + "Found %d instances of %s in response (expected %d)"
-                % (real_count, text_repr, count),
+                + "Found %d instances of %s in %s (expected %d)"
+                % (real_count, text_repr, content, count),
```

Comment:
I'm testing this out and am getting assertion failure messages like this:

> AssertionError: 0 != 1 : Found 0 instances of 'never' in b'No template used. Sample content: twice once twice. Content ends.' (expected 1)

Do we need to handle byte strings so we don't have the `b` show up?